sympy/tensor/array/ndim_array.py

```diff
@@ -586,6 +588,11 @@ def _check_index_for_getitem(self, index):
 
         return index
 
+    def _eval_evalf(self, prec=15):
```

Comment:
The default here should be `53` rather than `15` but actually it is just better not to provide any default because a default is not needed.

Reply:
The default has been removed. It was from an attempt to mitigate the errors earlier from the matrix class changes (and have parity between the matrix and array classes), but now it's back to the original suggested code.